app/boards/arm/preonic/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2020 The ZMK Contributors
```

Comment:
Please update all the copyright dates.

Reply:
updated

Reply:
Somehow your changes didn't show here.

Reply:
sorry, hadn't pushed. should be there now